perllib/FixMyStreet/DB/Result/User.pm

```diff
@@ -295,6 +295,15 @@ sub permissions {
 sub has_permission_to {
     my ($self, $permission_type, $body_ids) = @_;
 
+    # Nobody, including superusers, can have a permission which isn't available
+    # in the current cobrand.
+    if (my $cobrand = $self->result_source->schema->cobrand) {
```

Comment:
Shouldn't cobrand always be set if we've got here? It looks like only test failure was in `t/app/sendreport/inspection_required.t` in  https://travis-ci.org/mysociety/fixmystreet/jobs/270780037 so is the issue with the test? It trying to use a cobrand without overriding ALLOWED_COBRANDS, maybe?

Reply:
I saw a failure in `t/app/view/email.t`locally and so added that check, but yes I'll look into whether the test itself is wrong.

Reply:
Sorted, one of the tests was calling`has_permission_to` directly without a cobrand ever having been set up.